src/Integration.Vsix/CFamily/CFamilyIssueToAnalysisIssueConverter.cs

```diff
@@ -122,7 +122,7 @@ public IAnalysisIssue Convert(Message cFamilyIssue, string sqLanguage, ICFamilyR
                 severity: Convert(defaultSeverity),
                 type: Convert(defaultType),
 
-                filePath: cFamilyIssue.Filename,
+                filePath: Path.IsPathRooted(cFamilyIssue.Filename) ? Path.GetFullPath(cFamilyIssue.Filename) : cFamilyIssue.Filename,
```

Comment:
Similar to what we do in [secondary locations](https://github.com/SonarSource/sonarlint-visualstudio/blob/e1f9b8308698fd1133a405835d784304bd0ddfba/src/Integration.Vsix/CFamily/CFamilyIssueToAnalysisIssueConverter.cs#L160)